source/features/faster-pr-diff-options.tsx

```diff
@@ -5,6 +5,8 @@ import {BookIcon, CheckIcon, DiffIcon} from '@primer/octicons-react';
 
 import features from '.';
 
+const prUIBreakpointClassses = pageDetect.isPRFiles() ? 'hide-sm hide-md hide-lg' : '';
```

Comment:
Global constants aren't updated when the page changes via AJAX

Reply:
Also I think it should be the opposite: hide always, then `d-block-whatever the visible size is`